src/methods/burning/allocs.jl

```diff
@@ -19,7 +20,7 @@ function _burning_allocs(x;
 ) 
     if threaded
         if isnothing(x)
-            [Allocs(nothing) for _ in 1:nthreads]
```

Comment:
What prompted this?

Reply:
The nothing arg isn't needed now.

It worked before I'm just making it consistent with another change